src/styles/index.scss

```diff
@@ -1,3 +1,99 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding-inline: 16px;
+
+  &__image {
+    width: 160px;
+    padding-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+  span {
+    font-size: 10;
```

Comment:
The font-size property value should have a unit, like 'px'. Without a unit, this value is invalid and won't be applied. Make sure to add the appropriate unit.